src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/ReasonableAdjustmentsReportITest.java

```diff
@@ -0,0 +1,536 @@
+                                        .name("Main Phone")
+                                        .dataType("String")
+                                        .headings(null)
+                                        .build(),
+                                    StandardReportResponse.TableData.Heading.builder()
+                                        .id("other_phone")
+                                        .name("Other Phone")
+                                        .dataType("String")
+                                        .headings(null)
+                                        .build(),
+                                    StandardReportResponse.TableData.Heading.builder()
+                                        .id("work_phone")
+                                        .name("Work Phone")
+                                        .dataType("String")
+                                        .headings(null)
+                                        .build(),
+                                    StandardReportResponse.TableData.Heading.builder()
+                                        .id("email")
+                                        .name("Email")
+                                        .dataType("String")
+                                        .headings(null)
+                                        .build())
+                                ).build(),
+                            StandardReportResponse.TableData.Heading.builder()
+                                .id("next_attendance_date")
+                                .name("Next attendance date")
+                                .dataType("LocalDate")
+                                .headings(null)
+                                .build(),
+                            StandardReportResponse.TableData.Heading.builder()
+                                .id("juror_reasonable_adjustment_with_message")
+                                .name("Reasonable Adjustments")
+                                .dataType("List")
+                                .headings(List.of(
+                                    StandardReportResponse.TableData.Heading.builder()
+                                        .id("reasonable_adjustment_code_with_description")
+                                        .name("Reasonable Adjustment Code With Description")
+                                        .dataType("String")
+                                        .headings(null)
+                                        .build(),
+                                    StandardReportResponse.TableData.Heading.builder()
+                                        .id("juror_reasonable_adjustment_message")
+                                        .name("Juror Reasonable Adjustment Message")
+                                        .dataType("String")
+                                        .headings(null)
+                                        .build())
+                                ).build()))
```

Comment:
It would make sense to abstract out all of the boiler plate into its own method where you just provide the values that change i.e the new GroupedTableData() response payload and/or any headers that change (Keeps the code cleaner)